### PR TITLE
refactor(vps): remove uib modal

### PR DESCRIPTION
**Pull request by varun257**
Remove uib modal in vps

MANAGER-4092
Signed-off-by: Varun <varun257@gmail.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report installation#8276.0 ■   
* Stage 1
  * Install ■

**Comment by antleblanc** [comment]
Hi @varun257!

We recently merged #1336 which apply Prettier rules so could you please rebase
your branch on `develop`?

To easily resolve conflicts you can **ignore** `HEAD` changes and only keep yours.

After that you can run:

```sh
$ yarn run lint
```

This command will lint the entire codebase.

or:

Amend your commit to trigger the `pre-commit` hook (prettier will be launched
under the hood).

Thanks!

File: packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.html [hunk 1]
```diff
@@ -5,63 +5,40 @@
...
-                    <select
-                        name="access-ip"
-                        id="access-ip"
-                        class="oui-select__input"
-                        multiple
-                        required
-                        data-ng-options="ip for ip in $ctrl.available track by ip"
-                        data-ng-model="$ctrl.model.ip"
-                    >
+                    <select name="access-ip" id="access-ip" class="oui-select__input" multiple required
```
Comment (marie-j) [inline packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.html]: It seems like the prettier configuration was removed. 
You might want to try 
```suggestion
    prettier --write "packages/**/*.html"
```

File: packages/manager/modules/vps/src/veeam/vps-veeam.html [hunk 2]
```diff
@@ -35,12 +35,16 @@
...
-                    data-ng-if="$ctrl.veeam.data.accessInfos.restorePoint"
-                    data-ng-click="$ctrl.goToVeeamMount($ctrl.veeam.data.accessInfos.restorePoint, false)"
-                    data-ng-diabled="$ctrl.loaders.polling">
-                <span data-translate="vps_configuration_veeam_umount_title_button"></span>
+            <button
+                class="oui-button oui-button_secondary"
+                type="button"
+                data-ng-if="$ctrl.veeam.data.accessInfos.restorePoint"
+                data-ng-click="$ctrl.goToVeeamMount($ctrl.veeam.data.accessInfos.restorePoint, false)"
+                data-ng-diabled="$ctrl.loaders.polling"
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/veeam/vps-veeam.html]: Probably doesn't work

😄

Comment (FredericEspiau) [reply packages/manager/modules/vps/src/veeam/vps-veeam.html]: Couldn't you use a `oui-button` altogether ?

File: packages/manager/modules/vps/src/veeam/modal/restore/vps-restore.html [hunk 3]
```diff
@@ -5,26 +5,41 @@
...
+        <span
+            class="text-danger"
+            data-translate="vps_configuration_veeam_restore_step1_warning_time"
+        ></span>
+        <span
+            data-ng-if="$ctrl.attachedBackup"
+            class="text-danger"
+            data-translate="vps_configuration_veeam_mount_step1_warning_umount"
+        ></span>
     </p>
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/veeam/modal/restore/vps-restore.html]: You could probably have the `class` on the `p`

File: packages/manager/modules/vps/src/veeam/modal/mount/vps-mount.html [hunk 4]
```diff
@@ -6,20 +6,29 @@
...
     <dl class="oui-modal__text">
-        <dt data-translate="vps_configuration_veeam_restore_step1_creation"></dt>
+        <dt
+            data-translate="vps_configuration_veeam_restore_step1_creation"
+        ></dt>
         <dd data-ng-bind="$ctrl.restorePointLabel"></dd>
     </dl>
     <p class="oui-modal__text" data-ng-if="$ctrl.attachedBackup">
-        <span class="text-danger" data-translate="vps_configuration_veeam_mount_step1_warning_umount"></span>
+        <span
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/veeam/modal/mount/vps-mount.html]: Could be in the `p`

File: packages/manager/modules/vps/src/secondary-dns/vps-secondary-dns.html [hunk 5]
```diff
@@ -4,15 +4,31 @@
...
-                data-translate="vps_configuration_secondarydns_add_title_button"
-                data-ng-click="$ctrl.goToAddSecondaryDns()"></button>
-        <oui-datagrid data-ng-show="!$ctrl.loaders.init"
-                      data-rows="$ctrl.secondaryDns.list.results">
-            <oui-column data-title="'vps_tab_SECONDARY_DNS_table_domain' | translate" data-property="domain" data-sortable></oui-column>
-            <oui-column data-title="'vps_tab_SECONDARY_DNS_table_ip' | translate" data-property="ipMaster" data-sortable></oui-column>
-            <oui-column data-title="'vps_tab_SECONDARY_DNS_table_date' | translate" data-property="creationDate" data-sortable>
+        <button
+            type="button"
+            class="oui-button oui-button_secondary"
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/secondary-dns/vps-secondary-dns.html]: Could be a `oui-button`

File: packages/manager/modules/vps/src/secondary-dns/vps-secondary-dns.html [hunk 6]
```diff
@@ -4,15 +4,31 @@
...
+            data-translate="vps_configuration_secondarydns_add_title_button"
+            data-ng-click="$ctrl.goToAddSecondaryDns()"
+        ></button>
+        <oui-datagrid
+            data-ng-show="!$ctrl.loaders.init"
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/secondary-dns/vps-secondary-dns.html]: Can be replaced with a `ngIf` ?

File: packages/manager/modules/vps/src/secondary-dns/vps-secondary-dns.html [hunk 7]
```diff
@@ -21,8 +37,10 @@
                 data-sortable
             ></oui-column>
             <oui-action-menu data-align="end" data-compact>
-                <oui-action-menu-item data-text="{{'vps_common_delete' | translate}}"
-                                      data-on-click="$ctrl.goToDeleteSecondaryDns($row)"></oui-action-menu-item>
+                <oui-action-menu-item
+                    data-text="{{'vps_common_delete' | translate}}"
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/secondary-dns/vps-secondary-dns.html]: Deprecated, should be replaced with a transclusion

File: packages/manager/modules/vps/src/dashboard/modal/reinstall/vps-reinstall.html [hunk 8]
```diff
@@ -1,79 +1,148 @@
...
+                >
+                    <option
+                        value=""
+                        disabled
+                        selected
+                        data-translate="vps_configuration_reinstall_step1_language"
+                    ></option>
                 </select>
-                <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
+                <i
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/reinstall/vps-reinstall.html]: Should be a `span`

File: packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.component.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import controller from './add-backup-storage.controller';
+import template from './add-backup-storage.html';
+
+export default {
+  bindings: {
+    goBackToDetails: '<',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.component.js]:
```suggestion
    goBack: '<',
```
This way, the day we change where the component is, you don't have to rename this ;)

File: packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.module.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+
+import routing from './add-backup-storage.routing';
+import addComponent from './add-backup-storage.component';
+
+const moduleName = 'vpsBackupStorageAddModule';
+
+angular
+  .module(moduleName, [])
+  .config(routing)
+  .component('vpsBackupStorageAddComponent', addComponent)
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.module.js]:
```suggestion
  .component('vpsBackupStorageAdd', addComponent)
```

File: packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.routing.js [hunk 11]
```diff
@@ -0,0 +1,11 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('vps.detail.backup-storage.add', {
+    url: '/add',
+    views: {
+      modal: {
+        component: 'vpsBackupStorageAddComponent',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.routing.js]:
```suggestion
        component: 'vpsBackupStorageAdd',
```

File: packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.component.js [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import controller from './delete-backup-storage.controller';
+import template from './delete-backup-storage.html';
+
+export default {
+  bindings: {
+    access: '<',
+    goBackToDetails: '<',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.component.js]:
```suggestion
    goBack: '<',
```

File: packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.controller.js [hunk 13]
```diff
@@ -0,0 +1,25 @@
+export default class DeleteBackupStorageCtrl {
+  /* @ngInject */
+  constructor($translate, CucControllerHelper, CucCloudMessage,
+    VpsService) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.VpsService = VpsService;
+    this.CucControllerHelper = CucControllerHelper;
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.controller.js]: Alphabetical order

File: packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.controller.js [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  cancel() {
+    this.goBackToDetails();
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.controller.js]:
```suggestion
    return this.goBack();
```

File: packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.module.js [hunk 15]
```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+import routing from './delete-backup-storage.routing';
+import deleteComponent from './delete-backup-storage.component';
+
+const moduleName = 'vpsBackupStoragedeleteModule';
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.module.js]:
```suggestion
const moduleName = 'vpsBackupStorageDeleteModule';
```

File: packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.module.js [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+
+angular
+  .module(moduleName, [])
+  .config(routing)
+  .component('vpsBackupStorageDeleteComponent', deleteComponent)
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.module.js]:
```suggestion
  .component('vpsBackupStorageDelete', deleteComponent)
```

File: packages/manager/modules/vps/src/backup-storage/add/index.js [hunk 17]
```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import addModule from './add-backup-storage.module';
+
+const moduleName = 'vpsBackupStorageAdd';
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/add/index.js]:
```suggestion
const moduleName = 'vpsBackupStorageAddModule';
```

File: packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.routing.js [hunk 18]
```diff
@@ -0,0 +1,17 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('vps.detail.backup-storage.delete', {
+    url: '/delete',
+    params: {
+      access: null,
+    },
+    views: {
+      modal: {
+        component: 'vpsBackupStorageDeleteComponent',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/delete-backup-storage.routing.js]:
```suggestion
        component: 'vpsBackupStorageDelete',
```

File: packages/manager/modules/vps/src/backup-storage/delete/index.js [hunk 19]
```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import deleteModule from './delete-backup-storage.module';
+
+const moduleName = 'vpsBackupStorageDelete';
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/index.js]:
```suggestion
const moduleName = 'vpsBackupStorageDeleteModule';
```

File: packages/manager/modules/vps/src/backup-storage/delete/index.js [hunk 20]
```diff
@@ -0,0 +1,13 @@
...
+
+angular.module(moduleName, [
+  'ui.router',
+  deleteModule,
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/delete/index.js]: Alphabetical order

File: packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.component.js [hunk 21]
```diff
@@ -0,0 +1,12 @@
+import controller from './edit-backup-storage.controller';
+import template from './edit-backup-storage.html';
+
+export default {
+  bindings: {
+    row: '<',
+    goBackToDetails: '<',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.component.js]:
```suggestion
    goBack: '<',
```

File: packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.controller.js [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+      cifs: this.row.cifs,
+    };
+  }
+
+  validateCheckboxes() {
+    return this.model.ftp || this.model.cifs || this.model.nfs;
+  }
+
+  cancel() {
+    this.goBackToDetails();
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.controller.js]:
```suggestion
    return this.goBack();
```

File: packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.controller.js [hunk 23]
```diff
@@ -0,0 +1,51 @@
+import has from 'lodash/has';
+
+export default class EditBackupStorageCtrl {
+  /* @ngInject */
+  constructor($translate, CucControllerHelper, CucCloudMessage,
+    VpsService) {
+    this.$translate = $translate;
+    this.CucControllerHelper = CucControllerHelper;
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.controller.js]: Alphabetical order

File: packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.module.js [hunk 24]
```diff
@@ -0,0 +1,14 @@
...
+
+import routing from './edit-backup-storage.routing';
+import editComponent from './edit-backup-storage.component';
+
+const moduleName = 'vpsBackupStorageEditModule';
+
+angular
+  .module(moduleName, [])
+  .config(routing)
+  .component('vpsBackupStorageEditComponent', editComponent)
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.module.js]:
```suggestion
  .component('vpsBackupStorageEdit', editComponent)
```

File: packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.routing.js [hunk 25]
```diff
@@ -0,0 +1,17 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('vps.detail.backup-storage.edit', {
+    url: '/edit',
+    params: {
+      row: null,
+    },
+    views: {
+      modal: {
+        component: 'vpsBackupStorageEditComponent',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/edit/edit-backup-storage.routing.js]:
```suggestion
        component: 'vpsBackupStorageEdit,
```

File: packages/manager/modules/vps/src/backup-storage/edit/index.js [hunk 26]
```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import editModule from './edit-backup-storage.module';
+
+const moduleName = 'vpsBackupStorageEdit';
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/edit/index.js]:
```suggestion
const moduleName = 'vpsBackupStorageEditModule';
```

File: packages/manager/modules/vps/src/backup-storage/edit/index.js [hunk 27]
```diff
@@ -0,0 +1,13 @@
...
+
+angular.module(moduleName, [
+  'ui.router',
+  editModule,
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/edit/index.js]: Alphabetical order

File: packages/manager/modules/vps/src/backup-storage/index.js [hunk 28]
```diff
@@ -1,12 +1,24 @@
...
+import deleteStorage from './delete';
+import editStorage from './edit';
+import password from './password';
 
 const moduleName = 'ovhManagerVpsBackupStorage';
 
 angular
-  .module(moduleName, [ovhManagerVpsBackupStorageOrder])
+  .module(moduleName, [
+    ovhManagerVpsBackupStorageOrder,
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/index.js]: àlphabetical order

also, you use routing, translations and ui-kit, add those dependencies too

File: packages/manager/modules/vps/src/backup-storage/password/index.js [hunk 29]
```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import passwordModule from './password-backup-storage.module';
+
+const moduleName = 'vpsBackupStoragePassword';
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/password/index.js]:
```suggestion
const moduleName = 'vpsBackupStoragePasswordModule';
```

File: packages/manager/modules/vps/src/backup-storage/password/index.js [hunk 30]
```diff
@@ -0,0 +1,13 @@
...
+
+angular.module(moduleName, [
+  'ui.router',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/password/index.js]: Order

No translations and ui-kit ?

File: packages/manager/modules/vps/src/backup-storage/password/password-backup-storage.component.js [hunk 31]
```diff
@@ -0,0 +1,11 @@
+import controller from './password-backup-storage.controller';
+import template from './password-backup-storage.html';
+
+export default {
+  bindings: {
+    goBackToDetails: '<',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/password/password-backup-storage.component.js]:
```suggestion
    goBack: '<',
```

File: packages/manager/modules/vps/src/backup-storage/password/password-backup-storage.module.js [hunk 32]
```diff
@@ -0,0 +1,14 @@
...
+
+import routing from './password-backup-storage.routing';
+import passwordComponent from './password-backup-storage.component';
+
+const moduleName = 'vpsBackupStoragePasswordModule';
+
+angular
+  .module(moduleName, [])
+  .config(routing)
+  .component('vpsBackupStoragePasswordComponent', passwordComponent)
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/password/password-backup-storage.module.js]:
```suggestion
  .component('vpsBackupStoragePassword', passwordComponent)
```

File: packages/manager/modules/vps/src/backup-storage/password/password-backup-storage.routing.js [hunk 33]
```diff
@@ -0,0 +1,14 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('vps.detail.backup-storage.password', {
+    url: '/password',
+    params: {
+      row: null,
+    },
+    views: {
+      modal: {
+        component: 'vpsBackupStoragePasswordComponent',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/password/password-backup-storage.routing.js]:
```suggestion
        component: 'vpsBackupStoragePassword',
```

File: packages/manager/modules/vps/src/backup-storage/vps-backup-storage.routing.js [hunk 34]
```diff
@@ -8,5 +8,12 @@ export default /* @ngInject */ ($stateProvider) => {
         component: component.name,
       },
     },
+    resolve: {
+      goToAddBackupStorage: /* @ngInject */ ($state) => () => $state.go('vps.detail.backup-storage.add'),
+      goToDeleteBackupStorage: /* @ngInject */ ($state) => (access) => $state.go('vps.detail.backup-storage.delete', { access }),
+      goToEditBackupStorage: /* @ngInject */ ($state) => (row) => $state.go('vps.detail.backup-storage.edit', { row }),
+      goToPassword: /* @ngInject */ ($state) => () => $state.go('vps.detail.backup-storage.password'),
+      goBackToDetails: /* @ngInject */ ($state) => () => $state.go('vps.detail.backup-storage'),
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/backup-storage/vps-backup-storage.routing.js]:
```suggestion
      goBack: /* @ngInject */ ($state) => () => $state.go('vps.detail.backup-storage'),
```

File: packages/manager/modules/vps/src/dashboard/modal/display-ips/index.js [hunk 35]
```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import displayIpsModule from './vps-display-ips.module';
+
+const moduleName = 'vpsDashboardDisplayIps';
+
+angular.module(moduleName, [
+  'ui.router',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/display-ips/index.js]: Order, translations and ui-kit ?

File: packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.component.js [hunk 36]
```diff
@@ -0,0 +1,11 @@
+import controller from './vps-display-ips.controller';
+import template from './vps-display-ips.html';
+
+export default {
+  bindings: {
+    goBackToDashboard: '<',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.component.js]:
```suggestion
    goBack: '<',
```

File: packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.controller.js [hunk 37]
```diff
@@ -0,0 +1,24 @@
+export default class VpsCtrl {
+  /* @ngInject */
+  constructor($translate, CucControllerHelper, CucCloudMessage,
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.controller.js]: Order

File: packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.controller.js [hunk 38]
```diff
@@ -0,0 +1,24 @@
...
+    this.ipsLoader = this.CucControllerHelper.request.getHashLoader({
+      loaderFunction: () => this.VpsService.getIps(this.serviceName)
+        .then((data) => { this.ips = data.results; })
+        .catch(() => this.CucCloudMessage.error(this.$translate.instant('vps_configuration_reversedns_fail'))),
+    });
+    return this.ipsLoader.load();
+  }
+
+  cancel() {
+    this.goBackToDashboard();
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.controller.js]:
```suggestion
    return this.goBack();
```

File: packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.routing.js [hunk 39]
```diff
@@ -0,0 +1,11 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('vps.detail.dashboard.display-ips', {
+    url: '/display-ips',
+    views: {
+      modal: {
+        component: 'vpsDashboardDisplayIpsComponent',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.routing.js]:
```suggestion
        component: 'vpsDashboardDisplayIps',
```

File: packages/manager/modules/vps/src/dashboard/modal/kvm/index.js [hunk 40]
```diff
@@ -0,0 +1,15 @@
...
+import '@uirouter/angularjs';
+
+import kvmModule from './vps-kvm.module';
+import ovhManagerVpsKvmNovnc from './novnc/novnc.module';
+
+const moduleName = 'vpsDashboardKvm';
+
+angular.module(moduleName, [
+  'ui.router',
+  kvmModule,
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/kvm/index.js]: Order, translations and ui-kit ?

File: packages/manager/modules/vps/src/dashboard/modal/kvm/index.js [hunk 41]
```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import kvmModule from './vps-kvm.module';
+import ovhManagerVpsKvmNovnc from './novnc/novnc.module';
+
+const moduleName = 'vpsDashboardKvm';
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/kvm/index.js]:
```suggestion
const moduleName = 'vpsDashboardKvmModule';
```

File: packages/manager/modules/vps/src/dashboard/modal/kvm/vps-kvm.component.js [hunk 42]
```diff
@@ -0,0 +1,12 @@
+import controller from './vps-kvm.controller';
+import template from './vps-kvm.html';
+
+export default {
+  bindings: {
+    goBackToDashboard: '<',
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/modal/kvm/vps-kvm.component.js]:
```suggestion
    goBack: '<',
```

**Comment by varun257** [comment]
@FredericEspiau 
I have both `index.js` and `module.js` in all modules.

This is the naming convention i have used:
`index`: `vpsBackupStorageAdd`,
`module` `vpsBackupStorageAddModule`
`component`: `vpsBackupStorageAddComponent`.

Can you point me to proper naming convention for all of these files ? So i can follow the same everywhere from now on ?

**Comment by FredericEspiau** [comment]
@varun257 hey ! The convention has been talked about in a tech talk a few months ago, you can find it in our internal documentation center (can't link it here 😅 )

Your naming convention is mostly good, I'm just wondering:
- what's the difference between `index` and `module` ? It looks like you are not doing lazy loading so I'm not sure what the difference is 🤔 
- your component must not end with `Component`, imagine if you use your component in html, it would mean you would have to write `<vps-backup-storage-add-component></vps-backup-storage-add-component>`, don't you agree that `<vps-backup-storage-add></vps-backup-storage-add>` is better ? 🙂

**Comment by varun257** [comment]
> @varun257 hey ! The convention has been talked about in a tech talk a few months ago, you can find it in our internal documentation center (can't link it here 😅 )
> 
> Your naming convention is mostly good, I'm just wondering:
> 
> * what's the difference between `index` and `module` ? It looks like you are not doing lazy loading so I'm not sure what the difference is 🤔
> * your component must not end with `Component`, imagine if you use your component in html, it would mean you would have to write `<vps-backup-storage-add-component></vps-backup-storage-add-component>`, don't you agree that `<vps-backup-storage-add></vps-backup-storage-add>` is better ? 🙂

It does make sense to name the component this way. Module and index both might not be required here, probably makes more sense if there is lazy loading involved. Will remove it, thanks.

**Comment by varun257** [comment]
Please take a look now, all comments have been taken care of

File: packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.controller.js [hunk 43]
```diff
@@ -1,17 +1,8 @@
-export default class VpsDisplayIpsCtrl {
+export default class VpsCtrl {
```
[the remark this answers is not in the document]
Comment (varun257) [reply packages/manager/modules/vps/src/dashboard/modal/display-ips/vps-display-ips.controller.js]: Sure i can :)

**Pull request merged**